src/java/boa/aggregators/Aggregator.java

```diff
@@ -155,4 +179,12 @@ public void setKey(final EmitKey key) {
 	public EmitKey getKey() {
 		return this.key;
 	}
+	
+	public int getVectorSize() {
```

Comment:
move to MLAggregator?

Reply:
I have moved all of them to MLAggregator.